resume.md

```diff
@@ -1,49 +1,49 @@
 
 ## EDUCATION
-**University of College** | B.S. Degree ... *MONTH 20XX*
-* **Coursework**: Data Structures, Algorithms, Upper Division CS Elective, Math Course, Statistics, Data Science, ETC.
+
+**University of California, Berkeley** | B.S. Electrical Engineering & Computer Sciences | GPA: 3.71 ... *FALL 2021*
+
+* **Coursework**: Data Structures, Algorithms, Machine Structures, Statistics & Probability, Linear Algebra, Multivariable Calculus, Optimization,
 
 ## SKILLS
-* **Languages**: Python, Java, C, SQL, ...
-* **Libraries/Frameworks**: Pandas, TensorFlow, PyTorch, ...
-* **Version Control**: Git, Bitbucket, Sourcetree, GithubDesktop
-* *...*
+
+* **Languages**: Python, Java, C, SQL, JavaScript, HTML, CSS, MATLAB, Arduino,
+* **Frameworks and Tools**: ReactJS, TCP/IP, Threading, Git, NumPy, Socket, Socket.IO, Tkinter,
 
 ## EXPERIENCE
-**Software Engineer**: Company | City, State ... *Month 20XX - Present*
-* Implemented XYZ Microservice using TECHNOLOGY to delivery PRODUCT to 123 Customers Globally
-* Architected CORE INFRASTRUCTURE LIBRARY used across X teams to BLAH BLAH
-* I did X for Y to accomplish Z
-* ...
 
-**Software Engineer Intern**: Company | City, State ... *Month 20XX - Month 20XX*
-* Conducted A/B Test using FRAMEWORK to test EXPERIENCE in order to increase METRIC resulting in NUMBER. 
-* Utilized DEBUG TECHNOLOGY to resolve recurring crashes (X% of sessions) in beta before hitting production..
-* ...
+**Undergraduate Researcher, Software/Hardware**: Embodied Dexterity Group| Berkeley, CA ... *November 2019 - Present*
+
+* Led **team of 3** in designing a physics engine and GUI in Python to interface with a prosthetic arm, simulating the arm's movement and returning touch-related feedback. Incorporated the Model/View/Controller (MVC) design pattern to organize physics, GUI, and communication methods.
+* Established TCP connection to LabVIEW program using Python's socket library. Prevented packet loss and out-of-order delivery during communication, ensuring that any movement with our prosthetic arm resulted in accurate simulation and feedback. Allowed for meaningful, confident observations from the collected data.
+* Utilized multithreading to run communication/physics and GUI processing in parallel, overcoming stutters in the GUI refresh rate. Enabled the GUI to run at a consistent 60 frames per second, allowing test subjects to focus more easily on the experiment and preventing a potential study limitation.
+
+**Lead Machinist**: FIRST Robotics Competition Team 687 | Carson, CA ... *June 2015 - May 2018*
+
+* Led machining operations for a **10 member team**. Trained a total of **20+ students** on G-Code programming and proper manufacturing techniques to help them grow as engineers.
+* Improved record-keeping by implementing a detailed digital logbook of all completed and pending operations to keep team members updated on goals and progress while speeding up production times.
```

Comment:
digital log book - like a Gant Chart?  did you follow some sort of agile framework for product management?  Did you run all hands / meetings?